pkg/devfile/parser/parse.go

```diff
@@ -95,11 +112,6 @@ func parseParent(d DevfileObj) error {
 		return err
 	}
 
-	err = parentData.OverrideEvents(d.Data.GetParent().Events)
```

Comment:
`OverrideEvents()` is unused now and we can remove it along with UpdateEvents() used by it